libs/eo/auth/data-access/auth.interceptor.ts

```diff
@@ -39,7 +39,11 @@ export class EoAuthorizationInterceptor implements HttpInterceptor {
   private transloco: TranslocoService = inject(TranslocoService);
 
   private tokenRefreshCalls = ['PUT', 'POST', 'DELETE'];
-  private ignoreTokenRefreshUrls = ['/api/auth/token', '/api/authorization/consent/grant'];
+  private ignoreTokenRefreshUrls = [
+    '/api/auth/token',
+    '/api/authorization/consent/grant',
```

Comment:
Should this ('/api/authorization/consent/grant') be removed? I read the changes as this will be handled with the new polling mechanism.

Reply:
We have a general functionality which refreshes the token on API requests, but because we’re now polling for terms we ignore the request here to avoid an unneeded token refresh